[2025-02-08T19:03:22.069+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T19:03:22.094+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-02-08T19:03:22.102+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-02-08T19:03:22.103+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T19:03:22.210+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2025-01-30 00:00:00+00:00
[2025-02-08T19:03:22.220+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2136) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T19:03:22.221+0000] {standard_task_runner.py:72} INFO - Started process 2165 to run task
[2025-02-08T19:03:22.221+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2025-01-30T00:00:00+00:00', '--job-id', '387', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmpj9_ddbwn']
[2025-02-08T19:03:22.223+0000] {standard_task_runner.py:105} INFO - Job 387: Subtask generate_customer_dim_data
[2025-02-08T19:03:22.275+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-30T00:00:00+00:00 [running]> on host 2c1cd59afcc0
[2025-02-08T19:03:22.362+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-30T00:00:00+00:00'
[2025-02-08T19:03:22.363+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T19:03:22.382+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T19:03:22.383+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T19:03:22.391+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T19:03:22.392+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2025-01-30T00:00:00+00:00, execution_date=20250130T000000, start_date=20250208T190322, end_date=20250208T190322
[2025-02-08T19:03:22.437+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T19:03:22.454+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T19:03:22.456+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:43:49.513+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:43:49.531+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-02-08T20:43:49.540+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-02-08T20:43:49.541+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:43:49.555+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2025-01-30 00:00:00+00:00
[2025-02-08T20:43:49.564+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=886) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:43:49.565+0000] {standard_task_runner.py:72} INFO - Started process 896 to run task
[2025-02-08T20:43:49.565+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2025-01-30T00:00:00+00:00', '--job-id', '520', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmpm9nruu4f']
[2025-02-08T20:43:49.567+0000] {standard_task_runner.py:105} INFO - Job 520: Subtask generate_customer_dim_data
[2025-02-08T20:43:49.609+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-01-30T00:00:00+00:00 [running]> on host 86932746817d
[2025-02-08T20:43:49.690+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-30T00:00:00+00:00'
[2025-02-08T20:43:49.691+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:43:49.709+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T20:43:49.709+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:43:49.717+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:43:49.717+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2025-01-30T00:00:00+00:00, execution_date=20250130T000000, start_date=20250208T204349, end_date=20250208T204349
[2025-02-08T20:43:49.741+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:43:49.757+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:43:49.759+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
